x/datadeal/keeper/deal_test.go

```diff
@@ -617,3 +617,58 @@ func (suite dealTestSuite) TestDataDeliveryVoteFaildInvalidStatus() {
 	err = suite.DataDealKeeper.VoteDataDelivery(ctx, dataDeliveryVote, signature)
 	suite.Require().ErrorIs(err, types.ErrDataDeliveryVote)
 }
+
+func (suite dealTestSuite) TestDeactivateDeal() {
```

Comment:
I think it would be better to check the remained budget is refunded to the buyer properly.

Reply:
Fixed in [76fdb95](https://github.com/medibloc/panacea-core/pull/453/commits/76fdb95241636f7c7a0520c3013919322f3b57e4).